fre/cmor/cmor_mixer.py

```diff
@@ -311,10 +322,11 @@ def rewrite_netcdf_file_var ( proj_table_vars = None,
                                        axis_ids       = [cmor_lev, ],
                                        zfactor_values = ds["b_bnds"][:],
                                        units          = ds["b_bnds"].units )
-                cmor_lev = cmor.axis( "alternate_hybrid_sigma_half",
-                                      coord_vals = lev[:],
-                                      units = lev.units )
             else:
+                cmor_lev = cmor.axis( "alternate_hybrid_sigma",
+                                      coord_vals  = lev[:],
+                                      units       = lev.units,
+                                      cell_bounds = ds[vert_dim+"_bnds"] )
```

Comment:
Is this line creating the cell bounds from scratch or do they have to be in the input file?

If it's creating it from thin air, then this should be fine. But if it has to be in the input file, then perhaps it is named something different, but is probably `vert_dim+'_bnds'`. The CF way to find it would be similar to the dimension axis:

	double lev(lev) ;
		lev:units = "1.0" ;
		lev:long_name = "hybrid sigma pressure coordinate" ;
		lev:axis = "Z" ;
		lev:positive = "down" ;
		lev:formula = "p(n,k,j,i) = ap(k) + b(k)*ps(n,j,i)" ;
		lev:formula_terms = "ap: ap b: b ps: ps" ;
		lev:bounds = "lev_bnds" ;
		lev:standard_name = "atmosphere_hybrid_sigma_pressure_coordinate" ;

Reply:
`ds` is grabbed as a `netCDF4.Dataset` from the targeted files in the input directory, so `ds[vert_dim+'_bnds']` demands a value pointed to by key `vert_dim + '_bnds'` exists in `ds`.

Reply:
#241